.github/workflows/swift-toolchain.yml

```diff
@@ -1653,8 +1653,8 @@ jobs:
       - uses: actions/checkout@v4
         if: matrix.os == 'Android'
         with:
-          repository: apple/swift-corelibs-foundation
-          ref: a515f3e15c81d029411c540df7edf52d427ff2cb
+          repository: hyp/swift-corelibs-foundation
+          ref: 6bf677693eb23030eb3646c9b121c665f40aa8b0
```

Comment:
Is this for the changes wrt modulemaps?

Reply:
it's dropping the `fts_open` changes we had for Android after the posix_fielsystem api note fix landed (https://github.com/swiftlang/swift/pull/75494), since now Swift applies the api note to the `fts_open` API correctly in the workflow.

Reply:
I wonder if we can get the changes in upstream. I suppose that this is fine for now.

Reply:
Yeah this will all go away once recore is enabled in the workflow.